RPi3-installation.md

```diff
@@ -0,0 +1,28 @@
+# Installing LibreElec, KODI, and Netflix on a Raspberry Pi
+
+These instructions assume you have an unused Raspberry Pi and are starting from scratch. The steps will wipe whatever is on the Raspberry Pi's SD card.
+
+This has been tested on a Raspberry Pi 3, but the steps are likely to be very similar if not the same for other models.
+
+## Installation the latest Alpha LibreElec and KODI 18
+
+* Download the [LibreElec USB-SD Creator](https://libreelec.tv/downloads/)
+* Install the latest LibreElec stable version onto your Raspberry Pi's SD card using the USB-SD Creator 
+* Insert the SD card into the Raspberry Pi and boot it
+* Enable SSH and Samba (SMB) options when asked - make note of the default user and password
+* Download the [latest LibreElec Alpha](https://forum.kodi.tv/showthread.php?tid=298461) (These instructions were tested using 0722, but the latest build should generally work)
+* Connect to the Updates folder on the Raspberry Pi over SMB
+* Copy the LibreElec Alpha .tar file to the Updates folder on the Raspberry Pi
+* Reboot the Raspberry Pi - it should detect and install the alpha version
+* Enable SSH and Samba (SMB) options when asked again
+* Download the .zip of https://github.com/asciidisco/plugin.video.netflix from GitHub (click "Clone or Download" then "Download ZIP")
+* Connect to the Downloads folder on the Raspberry Pi over SMB
+* Copy the .zip file to the Downloads folder on the Raspberry Pi
+* On KODI go to Add-ons, Install from zip file
+* Navigate to the Downloads folder and select the zip file and install it
+* Connect to the Raspberry Pi over SSH using the default user and password
+* Run `wget http://nmacleod.com/public/libreelec/getwidevine.sh`
+* Run `sh getwidevine.sh`
```

Comment:
@jet86 Great work. Thank you.

The only thing that bugs me a little is the `getwidevine.sh` part. I´ve read a lot about what is allowed in terms of obtaining the widevine library (mostly because I wanted to implement an easier, non ssh, way to install it), and, lets put it that way, using the bash script mentioned above isn't a way Google would approve. That´s why I´m struggeling with adding this as the preferred way to the repo.

@MilhouseVH Any comments on that, you´re dealing with that stuff way longer than I, so you might have a more profound insights.